StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Views/CircularButton.swift

```diff
@@ -103,7 +103,7 @@ class CircularButton: UIControl {
             accessibilityIdentifier = "CircularButton.Remove"
         case .edit:
             imageView.image = Image.icon_edit.makeImage(template: true)
-            accessibilityLabel = String.Localized.update_card_brand
```

Comment:
I was wrong here, sorry! I got this mixed up w/ accessibilityIdentifier! Let's localize this, apologies for the churn on this.